_1327/shortlinks/templates/shortlinks_index.html

```diff
@@ -78,5 +81,38 @@
 				});
 			});
 		});
+
+		var shortlinkURLButton = $('.shortlink-copy');
+
+		shortlinkURLButton.popover({
+			content: '{% trans "Copied to clipboard." %}',
+			placement: 'top',
+			trigger: 'focus'
+		});
+
+		shortlinkURLButton.on('click', function (event) {
+			var urlHolder = $(this).parent().find(".shortlink-urltitle");
+			var baseURL = urlHolder.html();
+			var fullURl = document.location.origin + '/' + baseURL;
+
+			// create dummy textarea, paste the text to copy and copy that text
+			var el = document.createElement('textarea');
+			el.value = fullURl;
+			el.setAttribute('readonly', '');
+			el.style.position = 'absolute';
```

Comment:
why do you need the positioning - can't you just hide the element? i would actually prefer adding a hidden element on the page with the full url in the template and copying from there.

Reply:
we need the positioning, because otherwise the display might flicker shortly, once we add the element. We can not add a hidden element with the full url, because we don't know the full url on page creation, that's why we need Javascript to do that.